libs/ui/breadcrumb/helm/src/lib/breadcrumb.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'hlm-breadcrumb',
+  standalone: true,
+  template: `
+    <nav aria-label="breadcrumb">
+      <ng-content />
+    </nav>
+  `,
+})
+export class HlmBreadcrumbComponent {}
```

Comment:
I would recommend following the pattern used for pagination here as well.

Change it to an directive which applies the `aria-label` to a `nav` element

```html
<nav hlmBreadcrumb>
  ...
</nav>
```

Add an ariaLabel input for translation as well, similar to `HlmPaginationDirective`

https://github.com/spartan-ng/spartan/blob/95292eee9ba446fa58b4f1e2590c8119df284c7e/libs/ui/pagination/helm/src/lib/hlm-pagination.directive.ts#L12-L24

Reply:
I changed `HlmBreadcrumbComponent` to `HlmBreadcrumbDirective` like your suggestion.
Thanks